Cargo.toml

```diff
@@ -23,7 +23,7 @@ spec = "internal/config_specification.toml"
 [dependencies]
 anyhow = "1.0"
 bitcoin = { version = "0.29.2", features = ["serde", "rand"] }
-bitcoincore-rpc = "0.16"
+bitcoincore-rpc = { git = "https://github.com/rust-bitcoin/rust-bitcoincore-rpc", rev = "bde02d7fbf031df7d3a49946ec0e7f1abde34e58" }
```

Comment:
Update after https://github.com/rust-bitcoin/rust-bitcoincore-rpc/pull/253 is merged

Reply:
I vendored https://github.com/rust-bitcoin/rust-bitcoincore-rpc/compare/master...romanz:rust-bitcoincore-rpc:2023-02/electrs-build and released:
 - https://crates.io/crates/electrs-bitcoincore-rpc-json
 - https://crates.io/crates/electrs-bitcoincore-rpc